src/test/java/teammates/test/cases/logic/CoursesLogicTest.java

```diff
@@ -1064,13 +1066,21 @@ public void testCreateCourseAndInstructor() throws Exception {
         
         CourseAttributes invalidCourse = new CourseAttributes("invalid id", "Fresh course for tccai", "InvalidTimeZone");
         
+        String expectedError =
+                getPopulatedErrorMessage(
+                    FieldValidator.COURSE_ID_ERROR_MESSAGE, invalidCourse.getId(),
+                    FieldValidator.COURSE_ID_FIELD_NAME, FieldValidator.REASON_INCORRECT_FORMAT,
+                    FieldValidator.COURSE_ID_MAX_LENGTH) + EOL
+                + getPopulatedErrorMessage(
+                      FieldValidator.COURSE_TIME_ZONE_ERROR_MESSAGE, invalidCourse.getTimeZone(),
+                      FieldValidator.COURSE_TIME_ZONE_FIELD_NAME, FieldValidator.REASON_UNAVAILABLE_AS_CHOICE);
```

Comment:
@damithc Won't changing it back to magic strings beat the point of this issue - which asks to replace magic strings with constants?